app/main.py

```diff
@@ -1,13 +1,25 @@
+
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "failed" if errors else "passed"
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        format_single_linter_file(
+            file, linter_report[file]
+        ) for file in linter_report
```

Comment:
everything that inside return list is formatted in pretty strange way. You may use this tips to follow best practices: https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md

Or you can even use black package to make it automatically, docs: https://pypi.org/project/black/

Reply:
Fixed!